<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1001</width>
     <height>801</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#widget{
background-color:qlineargradient(spread:pad, x1:0.103182, y1:0.164, x2:1, y2:1, stop:0.0746269 rgba(161, 12, 40, 255), stop:1 rgba(255, 255, 255, 255))}</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>90</y>
      <width>291</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
margin-left: 15px;
font: italic 50px &quot;Rockwell&quot;;


</string>
    </property>
    <property name="text">
     <string>WELCOME</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>180</y>
      <width>361</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
font: 15pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Sign In or Create a new account</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>360</y>
      <width>291</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 15px;
font: 14pt &quot;MS Shell Dlg 2&quot;;
background-color:rgb(85, 255, 255);</string>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
   </widget>
   <widget class="QPushButton" name="createAccountButton">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>450</y>
      <width>291</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 15px;
font: 14pt &quot;MS Shell Dlg 2&quot;;
background-color:rgb(85, 255, 255);</string>
    </property>
    <property name="text">
     <string>Create a new Account</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
